rubygem-smart_proxy_dynflow/rubygem-smart_proxy_dynflow.spec

```diff
@@ -54,6 +54,9 @@ This package contains documentation for rubygem-%{gem_name}.
 %install
 mkdir -p %{buildroot}%{gem_dir}
 
+mkdir -p %{buildroot}%{_localstatedir}/lib/foreman-proxy/dynflow
+chown foreman-proxy:foreman-proxy %{buildroot}%{_localstatedir}/lib/foreman-proxy/dynflow
```

Comment:
Ownership should go into the %files list using %attr(...), as the foreman-proxy user ID won't be known during build (and may change).